app/forms/state_file/ny_sales_use_tax_form.rb

```diff
@@ -0,0 +1,32 @@
+module StateFile
+  class NySalesUseTaxForm < QuestionsForm
+    set_attributes_for :intake,
+                       :untaxed_out_of_state_purchases,
+                       :sales_use_tax_calculation_method,
+                       :sales_use_tax
+
+    validates :untaxed_out_of_state_purchases, presence: true
+    validates :sales_use_tax_calculation_method, presence: true, if: -> { untaxed_out_of_state_purchases == "yes" }
+    validates :sales_use_tax, presence: true, if: -> { sales_use_tax_calculation_method == "manual" }
+
+    def initialize(intake = nil, params = nil)
+      if params[:untaxed_out_of_state_purchases] == "no"
+        params[:sales_use_tax_calculation_method] = "unfilled"
+        params[:sales_use_tax] = nil
+      end
+      if params[:sales_use_tax_calculation_method] == "automated"
+        params[:sales_use_tax] = intake.calculate_sales_use_tax
+      end
+      super(intake, params)
+    end
+
+    def save
+      attributes = attributes_for(:intake)
+      attributes[:untaxed_out_of_state_purchases] = "unfilled" unless attributes[:untaxed_out_of_state_purchases].present?
+      attributes[:sales_use_tax_calculation_method] = "unfilled" unless attributes[:sales_use_tax_calculation_method].present?
+      attributes[:sales_use_tax] = intake.calculate_sales_use_tax if attributes[:sales_use_tax_calculation_method] == :automated
```

Comment:
I thought that perhaps this logic is handled in the model's before_save hook. The form_spec still passes if these are deleted, but it doesn't look like there are tests that would catch if this or the model didn't adjust other attributes based on the saved value.

Reply:
I feel confident that the model's before_save hook should take care of this. the first two are default values, so if they aren't present when you save the model, they'll be set to unfilled. If I understand the feature correctly, we only need to calculate sales use tax when this gets changed or initial set as "automated". I think this will calculate the sales use tax twice (in the form.save, and in the model.before_save) every time.

Reply:
Also, the validation should prevent anyone from saving with `untaxed_out_of_state_purchases = nil`

Reply:
Oh you're right! thanks for catching this, I think I added this in before adding the model validations and forgot to take it out